.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
```

Comment:
I'm personally confused as to when to use `*` and `**`. In the hello world style example on http://editorconfig.org they show `[*]` as a sort of default setting for all files, but they also explain how * does not include slashes. So wouldn't that mean that this rule will match only files in the root dir? That would be a very odd default, so I must be mistaken. I just can't find an explanation for it.